Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeInformation.ps1

```diff
@@ -236,6 +244,15 @@ function Get-ExchangeInformation {
         }
 
         [array]$serverMonitoringOverride = Get-MonitoringOverride -Server $Server
+
+        if ($null -ne $ianaTimeZoneMappingContent) {
+            try {
+                [xml]$ianaTimeZoneMappingXml = $ianaTimeZoneMappingContent
+            } catch {
+                Write-Verbose "Unable to convert IanaTimeZoneMappings.xml to Xml"
+                Invoke-CatchActions
+            }
```

Comment:
Let's move this to analysis, would rather it be stored in the object if there is a failure, we want to see why that is from the raw content.

Reply:
Done